### ISSUE TITLE
Fix latest fluentd image 

### ISSUE THREAD

**Issue by serut** [fluent/fluentd-docker-image#214]
Hello,

I'm working on a stack that needs to be run with read only + username space to follow our security guidelines.

Indeed your image creates an user to avoid running as root, but you do not allow running your entire container with an user provided using `--user "30XX:10XX"`. Of course I can fix it by removing the call to /bin/entrypoint.sh but I think your image would better to follow what others did, like postgres.

- if the user is root, make what you have already write
- if the user is unknown, just execute fluentd.

Do you have any consideration that prevents you to do the above pattern ? Why are you creating an user with an home directory ?

Current output :

```
adduser: permission denied (are you root?)
chown: unknown user fluent
chown: unknown user fluent
su-exec: setgroups(1401050): Operation not permitted
```
-> you should add a `set -e` in your script !!!

My workaround : 
```
FROM fluent/fluentd
[.. plugins install]
# as the container is read only, set the folder writtable by anyone
RUN chmod 0666 /fluentd/log

ENTRYPOINT ["sh"]
CMD ["fluentd", "-c", "/fluentd/etc/fluent.conf"]
```

Postgres entrypoint.sh : 
```
[.. some init]

# allow the container to be started with `--user`
if [ "$1" = 'postgres' ] && [ "$(id -u)" = '0' ]; then
	mkdir -p "$PGDATA"
	chown -R postgres "$PGDATA"
	chmod 700 "$PGDATA"

	mkdir -p /var/run/postgresql
	chown -R postgres /var/run/postgresql
	chmod 775 /var/run/postgresql

	# Create the transaction log directory before initdb is run (below) so the directory is owned by the correct user
	if [ "$POSTGRES_INITDB_WALDIR" ]; then
		mkdir -p "$POSTGRES_INITDB_WALDIR"
		chown -R postgres "$POSTGRES_INITDB_WALDIR"
		chmod 700 "$POSTGRES_INITDB_WALDIR"
	fi

	exec gosu postgres "$BASH_SOURCE" "$@"
fi
[... some postgres stuff run to init the db]
exec "$@"
```

Basically, `if [ "$1" = 'postgres' ] && [ "$(id -u)" = '0' ]; then` means "do something with user rights if the container is not started with specific `--user`

hope it helps to make it cleaner

**Comment by serut** [comment]
Ok nethermind. I switch from latest to v1.10-1 and all was fixed. You should remove / fix the latest version of your image !!!!

**Comment by repeatedly** [comment]
`latest` is for backward compatibility. See README: https://github.com/fluent/fluentd-docker-image#for-older-images

**Comment by serut** [comment]
Yeah but you should add some log in the `docker-entrypoint.sh` container to brings users to use your postfix version : `Fluentd is not using the "latest" version anymore, in favor to postfix version. See : https://github.com/fluent/fluentd-docker-image`

This looks way to complicated actually. I installed maybe 8 components around our stack and you're the only one that made this decision (latest => outdated).
